src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public abstract void drawFigure();
```

Comment:
it would be better if we move this method to interface (we can create new one)

Reply:
Ok) andestand. I fix it